packages/components/src/ConfirmationModal/ConfirmationModal.mdx

```diff
@@ -40,6 +42,7 @@ import {
           onConfirm={() => alert("✅")}
           onCancel={() => alert("🙅‍♂️")}
           onRequestClose={() => setOpen(false)}
+          type="work"
```

Comment:
This can be left off as it is supplied with a default value of `work`

Reply:
removed